source/extensions/filters/http/jwt_authn/authenticator.cc

```diff
@@ -82,7 +82,8 @@ class AuthenticatorImpl : public Logger::Loggable<Logger::Id::jwt>,
   std::unique_ptr<::google::jwt_verify::Jwt> jwt_;
   // The JWKS data object
   JwksCache::JwksData* jwks_data_{};
-
+  // The TokenCache object
+  std::unique_ptr<TokenCache> token_cache_{std::make_unique<TokenCache>()};
```

Comment:
Not, TokenCache should be inside JwksCache,  inside its CacheData class

Reply:
Done.